.shopify-build/polaris-react.yml

```diff
@@ -0,0 +1,45 @@
+containers:
+  default:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.11.0-yarn-1.12.3
+  styleguide:
+    docker: circleci/node:10.13.0
+
+shared:
+  build-web: &build-web
+    label: 'Build web'
+    container: default
+    timeout: 35m
+    dependencies:
+      - yarn
+    run:
+      - yarn run build
+      - ./.shopify-build/setup-consumer-test.sh web
+      - yarn --cwd ./web run build
+  build-styleguide: &build-styleguide
+    label: 'Build styleguide'
+    container: styleguide
+    timeout: 20m
+    dependencies:
+      - yarn
+    run:
+      - yarn run build
+      - ./.shopify-build/setup-consumer-test.sh polaris-styleguide
+      - yarn --cwd ./polaris-styleguide run build
+  unit-tests: &unit-tests
+    label: '🤓 Unit tests'
```

Comment:
"Web Unit tests" perhaps, as it's not clear if immediatly clear if they're for web or styleguide.

Related: Should we run the styleguide tests too?

Reply:
We could, but the styleguide tests are very spotty and I doubt they'd help much, they rarely fail. What do you think?

Reply:
Cool, happy to leave them out if you don't think they're worth it.